src/components/biggive-footer/biggive-footer.tsx

```diff
@@ -68,5 +76,155 @@ export class BiggiveFooter {
         </div>
       </footer>
     );
+    }
+
+    const presetFooter = () => {
+      return (
+        <footer class="footer">
```

Comment:
@dee-jo could you confirm / link to where the content below comes from? Thanks.

Reply:
I think we should make these link to the specific environmentally appropriate version of these URLs according which environment we're running in - e.g. share or copy the `blogUrl` and similar functions from the mainMenu component so that it can point to dev in dev, staging in staging, regression in regression, prod in prod. It's a bit confusing to have.

On the other hand we already have that mismatch in donate so this wouldn't be a regression and therefore shouldn't necessarily block this PR.

Reply:
blogUrl etc functions are simple enough personally I don't think it matters really whether we share them or copy and paste them between the two components
https://github.com/thebiggive/components/blob/54b4ab3eafa99e179d4373dd4772ad3fe6635bde/src/components/biggive-main-menu/biggive-main-menu.tsx#L186